akka-persistence-query/src/test/scala/akka/persistence/query/internal/QuerySerializerSpec.scala

```diff
@@ -24,8 +25,8 @@ class QuerySerializerSpec extends AkkaSpec {
     val serializer = serialization.findSerializerFor(obj).asInstanceOf[SerializerWithStringManifest]
     val manifest = serializer.manifest(obj)
     val bytes = serialization.serialize(obj).get
-    val deserialzied = serialization.deserialize(bytes, serializer.identifier, manifest).get
-    deserialzied shouldBe obj
+    val deserialized = serialization.deserialize(bytes, serializer.identifier, manifest).get
+    deserialized shouldBe obj
   }
 
   "Query serializer" should {
```

Comment:
note to self: the normal serializer of the EventEnvelope should use the eventBytes in toBinary

maybe it should also include the fact that it's a lazy envelope so in fromBinary it would recreate the envelope with the bytes and deserialize function